packages/react-ui-components/src/SelectBox/selectBox.js

```diff
@@ -5,7 +5,7 @@ import {$get} from 'plow-js';
 import SelectBox_Option_SingleLine from '../SelectBox_Option_SingleLine';
 import mergeClassNames from 'classnames';
 import isEqual from 'lodash.isequal';
-import isNil from '../../../utils-helpers/src/isNil';
+import {isNil} from '@neos-project/utils-helpers';
```

Comment:
The package manifest of `@neos-project/react-ui-components` is missing this dependency.

Reply:
Thx!